### ISSUE TITLE
8.5

### ISSUE THREAD

**Issue by vzaliva** [coq/opam-coq-shell#4]
Currently only 8.4 and dev profiles are offered. Shouldn't we have 8.5 as well?

**Comment by gares** [comment]
Absolutely, but the thing is not "ready yet".  I hope we will manage to work on it next week.

**Comment by clarus** [comment]
By the way, there should be a `released` profile as well, including only the `released` repository.
